src/App.tsx

```diff
@@ -52,60 +54,89 @@ const App = () => {
   };
 
   let routes;
+  if (userRole.length == 0) {
+    // on page refresh wait to get user role
+    setTimeout(() => {
+      setRole(getUserRole());
+    }, 1000);
+  }
 
   if (authenticated) {
     const defaultRedirect = () => <Redirect to="/chat" />;
-    routes = (
-      <div className={styles.App}>
-        <Layout>
-          <Switch>
-            <Route path="/tag" exact component={TagPage} />
-            <Route path="/tag/add" exact component={Tag} />
-            <Route path="/tag/:id/edit" exact component={Tag} />
-            <Route path="/speed-send" exact component={SpeedSendPage} />
-            <Route path="/speed-send/add" exact component={SpeedSend} />
-            <Route path="/speed-send/:id/edit" exact component={SpeedSend} />
-            <Route path="/automation" exact component={AutomationList} />
-            <Route path="/automation/add" exact component={Automation} />
-            <Route path="/automation/:id/edit" exact component={Automation} />
-            <Route path="/group" exact component={GroupList} />
-            <Route path="/group/add" exact component={Group} />
-            <Route path="/group/:id/edit" exact component={Group} />
-            <Route path="/group/:id/contacts" exact component={GroupContact} />
+    let route;
+    if (userRole.includes('Staff')) {
```

Comment:
I think we should move routes out of app.tsx, it's getting cluttered. We could potentially just get routes by role method included here.